### DISCUSSION TITLE
Redirect on 4433 to 4434 API caused me a bit of confusion

### DISCUSSION THREAD

**Discussion by Rodeoclash** [ory/kratos#3294] [Ideas]
I've misconfigured my internal API to point towards port 4433 on Kratos when making admin queries. However, this just works(tm)

After scratching my head a bit, I tried curl and got:

```
*   Trying 127.0.0.1:4433...
* TCP_NODELAY set
* Connected to localhost (127.0.0.1) port 4433 (#0)
> GET /admin/identities/1234 HTTP/1.1
> Host: localhost:4433
> User-Agent: curl/7.68.0
> Accept: */*
>
* Mark bundle as not supporting multiuse
< HTTP/1.1 307 Temporary Redirect
< Cache-Control: private, no-cache, no-store, must-revalidate
< Content-Type: text/html; charset=utf-8
< Location: http://ory-kratos:4434/admin/identities/1234
< Vary: Origin
< Vary: Cookie
< Date: Mon, 29 May 2023 11:18:48 GMT
< Content-Length: 80
<
<a href="http://ory-kratos:4434/admin/identities/1234">Temporary Redirect</a>.
```

So clearly Kratos is being helpful in redirecting my request to the right location (this is all over a private network, so I can access 4434 internally without issue.

However, I think it would be a good idea to capture this behaviour somewhere, perhaps on the https://www.ory.sh/docs/kratos/quickstart page? Happy to raise a PR with some guidance about where to capture this

**Comment by vinckr** [comment]
Hello @Rodeoclash
I am not quite sure how this behaviour works and it does not seem to be documented. 
Before we add something to the quickstart we probably should look at the code and confirm how it exactly works.
